fuse_variables/include/fuse_variables/orientation_3d_stamped.h

```diff
@@ -105,13 +97,9 @@ class Orientation3DLocalParameterization : public fuse_core::LocalParameterizati
     ceres::AngleAxisToQuaternion(delta, q_delta);
     ceres::QuaternionProduct(x, q_delta, x_plus_delta);
     return true;
-}
+  }
```

Comment:
Not sure if the indentation got wrong here: :thinking: 
```suggestion
}
```